.github/workflows/CI.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         key: cached_node_modules_${{ hashFiles('**/yarn.lock') }}
         restore-keys: cached_node_modules_
     - if: steps.cache-node-modules.outputs.cache-hit != 'true'
-      run: yarn --prefer-offline --frozen-lockfile --ignore-engines
+      run: yarn --prefer-offline --ignore-engines
```

Comment:
Don't we prefer frozen lockfiles? What prompted the change?

Reply:
But you added a new lock file to this PR. Shouldn't that have all the updates it needs?

Reply:
Ahh, I didn't push the updated `yarn.lock` after moving faker... Will rectify

Reply:
realized that a lot of the yarn.lock file changes came from when I updated ember-faker, which I didn't need to do, so force-pushed without that commit in there to keep things a bit easier to read